buildAndReleaseTask/Invoke-tSQLtTests-WithCodeCoverage.ps1

```diff
@@ -11,6 +11,7 @@ param (
     [string]$coberturaFileName,
     [string]$htmlReportsOutput,
     [string]$queryTimeout
```

Comment:
There may have a comma missing here, after $queryTimeout.

I think it breaks the code execution. But in case we're not sure, it would be good to keep the standard.